contrib/polaris/config.go

```diff
@@ -110,7 +111,7 @@ func (w *ConfigWatcher) Next() ([]*config.KeyValue, error) {
 		}
 		return w.cfg, nil
 	}
-	return w.cfg, nil
+	return w.cfg, context.Canceled
```

Comment:
Is it reasonable to return a context error

Reply:
I think it's reasonable because when the connection is already closed, it is indeed necessary to throw a context exception to the outer layer to cancel the file listener.